src/plugins/minimongo/inject.js

```diff
@@ -0,0 +1,22 @@
+import _ from 'underscore';
```

Comment:
would be great to exclude this so we don't inject the whole underscore into everyone's page (will have to do good ol' for(var i ... ))